src/renderer/src/routes/pages/Contacts/index.tsx

```diff
@@ -32,6 +32,10 @@ export const ContactsPage = () => {
 
   const [sortBlockchain, setSortBlockchain] = useState<boolean | null>(null)
 
+  useEffect(() => {
+    setSortBlockchain(null)
+  }, [selectedContact])
```

Comment:
Move this to the end of the component before the return, as it is easier to read and maintains a pattern in the components with hooks on top, then functions and then effects.